__tests__/feeds.ts

```diff
@@ -655,7 +655,7 @@ describe('query sourceFeed', () => {
     now = new Date(),
     first = 10,
   ): string => `{
-    sourceFeed(source: "${source}", ranking: ${ranking}, now: "${now.toISOString()}", first: ${first}) {
+    sourceFeed(source: "${source}", ranking: ${ranking}, now: "${now.toISOString()}", first: ${first}, supportedTypes: ["article", "share", "welcome"]) {
```

Comment:
> If we support freeForm won't it support both out of the box already?

Yes, it will be just a matter of adding another parameter for supported types.